samples/drivers/led_ws2812/boards/nrf7002dk_nrf5340.overlay

```diff
@@ -0,0 +1,41 @@
+#include <zephyr/dt-bindings/led/led.h>
+
+&pinctrl {
+	i2s0_default_alt: i2s0_default_alt {
+		group1 {
+			/* Default I2S config for the nRF5340, P1.13 is the output */
+			psels = <NRF_PSEL(I2S_SCK_M, 1, 15)>,
+				<NRF_PSEL(I2S_LRCK_M, 1, 12)>,
+				<NRF_PSEL(I2S_SDOUT, 1, 13)>,
+				<NRF_PSEL(I2S_SDIN, 1, 14)>;
+		};
+	};
+};
+
+i2s_led: &i2s0 {
+	status = "okay";
+	pinctrl-0 = <&i2s0_default_alt>;
+	pinctrl-names = "default";
+};
+
+&gpio1 {
+	status = "okay";
+};
```

Comment:
Why does `gpio1` need to be set to `okay` ?

Reply:
I remember I needed to set it for a project and I thought that since the I2S device is on `gpio1`, I needed it here as well. But I checked and it compiled without this so I removed these four lines.